src/components/molecules/Table/TableTasks.jsx

```diff
@@ -0,0 +1,81 @@
+import styled from "styled-components";
+import "reset-css";
+import { useState } from "react";
+import Button from "../../atoms/Button";
+import useProject from "../../../lib/hooks/states/project";
+import TableTrTasks from "../../atoms/table_tr_tasks";
+import ModalWindowCreateTask from "../../atoms/ModalCreateTask";
+
+const StyledTable = styled.table`
+  top: 0;
+  right: 100px;
```

Comment:
top and right doesnt work there, can be removed.
It can be used if container has  default `position: static;`

Reply:
ok, tnx!